website/docs/addon_resolve_admin.md

```diff
@@ -15,96 +15,51 @@ import Tabs from '@theme/Tabs';
 
-### Installing Resolve's own python 3.6 interpreter.
-Resolve uses a hardcoded method to look for the python executable path. All of tho following paths are defined automatically by Python msi installer. We are using Python 3.6.2.
+Resolve 17 is using Python 3.6.2
+Resolve 18 is using Python 3.9.x
 
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
 
-<TabItem value="win">
+### Configuring settings for Python home directory
+To be able to run python in Resolve you need to set `RESOLVE_PYTHON3_HOME` environment variable to the directory path of your Python interpreter version.
 
-`%LOCALAPPDATA%\Programs\Python\Python36`
+Open your server Studio settings and go to **Application** addon `ayon+settings://applications/applications/resolve`. Here you can add the path to the **Environment** section.
 
-</TabItem>
-<TabItem value="linux">
+![Ayon/Application/Resolve](assets/resolve_python_home_application_addon.png)
 
-`/opt/Python/3.6/bin`
 
-</TabItem>
-<TabItem value="mac">
+### Installing Python dependencies
 
-`~/Library/Python/3.6/bin`
+#### PySide2
 
-</TabItem>
-</Tabs>
-
-
-### Installing PySide2 into python 3.6 for correct gui work
-
-AYON is using its own window widget inside Resolve, for that reason PySide2 has to be installed into the python 3.6 (as explained above).
-
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
-
-<TabItem value="win">
-
-paste to any terminal of your choice
+AYON is using its own window widget inside Resolve, for that reason PySide2 (or PySide6 if on MacOS) has to be installed into the python.
 
 ```bash
-%LOCALAPPDATA%\Programs\Python\Python36\python.exe -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python.exe -m pip install PySide2
 ```
 
-</TabItem>
-<TabItem value="linux">
+### OpenTimelineIO
 
-paste to any terminal of your choice
+AYON is using OpenTimelineIO for editorial publishing. OpenTimelineIO has to be installed into the python.
 
 ```bash
-/opt/Python/3.6/bin/python -m pip install PySide2
-```
-
-</TabItem>
-<TabItem value="mac">
-
-paste to any terminal of your choice
 
 ```bash
-~/Library/Python/3.6/bin/python -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python.exe -m pip install OpenTimelineIO
 ```
 
-</TabItem>
-</Tabs>
-
-<div class="row markdown">
+### Debugging if Python is not working in Resolve
 
-### Set Resolve's Fusion settings for Python 3.6 interpereter
 
-<div class="col col--6 markdown">
+#### Set Resolve's Fusion settings for Python 3.6 interpreter
```

Comment:
```suggestion
#### Set Resolve's Fusion settings for Python 3 interpreter
```

This is Py3.9 recommended for Resolve 18 in this readme above. I think it'd be best to make this section about Py3 - not specifically Py3.6